apps/web/src/services/my-page/my-account/user-account/modules/MultiFactorAuthentication.vue

```diff
@@ -0,0 +1,171 @@
+<script setup lang="ts">
+import {
+    computed, reactive,
+} from 'vue';
+
+import {
+    PI, PTextInput, PFieldGroup, PToggleButton, PSelectDropdown, PFieldTitle,
+} from '@spaceone/design-system';
+
+import { store } from '@/store';
+import { i18n } from '@/translations';
+
+import { emailValidator } from '@/lib/helper/user-validation-helper';
+
+import { useFormValidator } from '@/common/composables/form-validator';
+import VerifyButton from '@/common/modules/button/verify-button/VerifyButton.vue';
+
+import UserAccountModuleContainer
+    from '@/services/my-page/my-account/user-account/modules/UserAccountModuleContainer.vue';
+
+// Currently, only email is supported.
+const contextMenuItems = [
+    { label: i18n.t('IDENTITY.USER.MFA.EMAIL'), name: 'email' },
+];
```

Comment:
suggestion: It seems like you should switch to computed!

![스크린샷 2023-11-14 오전 9 07 36](https://github.com/cloudforet-io/console/assets/18563857/d5eb0e49-32ce-4f78-b57e-a0f8154cc561)